requirements/doc-spelling.in

```diff
@@ -0,0 +1,2 @@
+-r doc.txt
+sphinxcontrib-spelling==7.1.0; platform_system!="Windows"  # We only use it in Travis CI
```

Comment:
Lol we switched two CI systems after Travis. But nevermind, the comment not vital for this PR.